tests/test_filters.py

```diff
@@ -118,3 +120,13 @@ def test_capitalize_first_for_non_strings():
     assert capitalize_first([{'list': 'of'}, 'things']) == [{'list': 'of'}, 'Things']
     assert capitalize_first({'this': 'thing'}) == {'this': 'thing'}
     assert capitalize_first('https://www.example.com') == 'https://www.example.com'
+
+
+def test_preserve_line_breaks():
```

Comment:
You _could_ use `pytest.mark.parametrize` to pass in different values for `autoescape`, just so we can see that the output is the same.